### PR TITLE
Align both reflectly-hero & dart-diagram to the center.

### PR DESCRIPTION
**Pull request by Pradumnasaraf**
**Aligned both reflectly-hero & dart-diagram to the center, so that it can look aligned w.r.t first image (Flutter logo) in README.md and also follow a similar pattern.**
---

**Before Changes**

Before Changes

**After Changes**

After Changes

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt.
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

**Comment by Pradumnasaraf** [comment]
Please see to this PR :).@justinmc @kjlubick @Piinks @dnfield.

File: README.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ graphics, video, text, and controls without limitation. Flutter includes a full
 [set of widgets][widget catalog] that deliver pixel-perfect experiences on both
 iOS and Android.
 
-![Reflectly hero image][Reflectly hero image]
+<p align = "center"><img src="https://github.com/flutter/website/blob/main/src/assets/images/docs/homepage/reflectly-hero-600px.png?raw=true"></p>
```
Comment (justinmc) [inline README.md:39]: Can you add the alt text? `alt="Reflectly hero image"`

Comment (justinmc) [reply README.md:39]: Also nit:

```diff
- align = "center"
+ align="center"
```

File: README.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ graphics, video, text, and controls without limitation. Flutter includes a full
 [set of widgets][widget catalog] that deliver pixel-perfect experiences on both
 iOS and Android.
 
-![Reflectly hero image][Reflectly hero image]
```
Comment (justinmc) [inline README.md:39]: Is it still possible to use the reference to `Reflectly hero image` in the `src` attribute?  Maybe not.

**Review by justinmc — COMMENTED** [review]
Thanks for the quick improvement!  Looks better to me.

I think we'll need an exemption for no tests, I'll post on Discord.

**Comment by Hixie** [comment]
test-exempt: only affects markdown

File: README.md (hunk 2 above)
Comment (Pradumnasaraf) [reply README.md:39]: No, we can't use it in that way

File: README.md (hunk 1 above)
Comment (Pradumnasaraf) [reply README.md]: @justinmc Done 

```html
<p align="center"><img src="https://github.com/flutter/website/blob/main/src/assets/images/docs/homepage/reflectly-hero-600px.png?raw=true" alt="Reflectly hero image"></p>

<p align="center"><img src="https://github.com/flutter/website/blob/main/src/assets/images/docs/homepage/dart-diagram-small.png?raw=true" alt="Dart diagram"></p>
```

**Comment by Pradumnasaraf** [comment]
@justinmc Done
```html
<p align="center"><img src="https://github.com/flutter/website/blob/main/src/assets/images/docs/homepage/reflectly-hero-600px.png?raw=true" alt="Reflectly hero image"></p>

<p align="center"><img src="https://github.com/flutter/website/blob/main/src/assets/images/docs/homepage/dart-diagram-small.png?raw=true" alt="Dart diagram"></p>
```

**Comment by Pradumnasaraf** [comment]
@justinmc looks like you forgot to merge :)

**Comment by Pradumnasaraf** [comment]
Sorry to ping you, @Piinks, @goderbauer, @justinmc if everything is okay please merge 😀

**Comment by Piinks** [comment]
> Sorry to ping you, @Piinks, @goderbauer, @justinmc if everything is okay please merge 😀

It has not been approved yet, @Pradumnasaraf, that is probably why. :)

**Review by Piinks — APPROVED** [review]
Flutter_LGTM

**Comment by Pradumnasaraf** [comment]
@Piinks  Thank You so much :). Can I put my name in AUTHOR if you don't mind (Also done for other Flutter Repo)?

**Pull request merged**